clients/flash/common-library/src/main/actionscript/org/bigbluebutton/lib/screenshare/services/ScreenshareMessageSender.as

```diff
@@ -0,0 +1,13 @@
+package org.bigbluebutton.lib.screenshare.services
+{
+
+	public class ScreenshareMessageSender

```

Comment:
Duplicate class. This one can be removed.

Reply:
Duplicate with which class? This class is specific for screenshare which connects to the screenshare red5 app and not to bigbluebutton red5 app. Do you want me to lump the functions in this class with which *Reciever?

Reply:
Deleted.